harness/determined/core/_tensorboard_mode.py

```diff
@@ -0,0 +1,14 @@
+import enum
+
+
+class TensorboardMode(enum.Enum):
+    """
+    ```TensorboardMode`` defines how Tensorboard metrics and profiling data are retained.
+    In ``Auto`` mode only the chief uploads its metrics and profiling data to checkpoint
+    storage. This is the same behavior that existed prior to 0.18.2.
+    In ``Manual`` mode neither metrics nor profiling data are reported. It is up to the user
+    to call ``TrainContext.upload_tensorboard_files()``
```

Comment:
You say "defines how Tensorboard **metrics and profiling** data are **retained**", but this applies to all tensorboard files of every kind (not just metrics and profiling), and it isn't really a retention policy.

Recommended change:
```
``TensorboardMode`` defines how Tensorboard artifacts are handled.

In ``Auto`` mode the chief automatically writes any reported training or validation 
metrics to the Tensorboard path (see meth:`core_context.trian.get_tensorboard_path<
determined.core.TrainContext.get_tensorboard_path>`), and automatically 
uploads all of its own tensorboard artifacts to checkpoint storage.  Tensorboard 
artifacts written by non-chief workers will not be uploaded at all. This is the 
same behavior that existed prior to 0.18.2.

In ``Manual`` mode no Tensorboard artifacts are written or uploaded at all. 
It is entirely up to the user to write their desired metrics and upload 
them with calls to :ref:`core_context.train.upload_tensorboard_files<
determined.core.TrainContext.upload_tensorboard_files>`.
```